app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -785,6 +789,16 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         }
     }
 
+    override fun getSystemService(name: String): Any? {
```

Comment:
So in what cases is the inflater used and which isn't it used? e.g. is it used for inflating Fragment views but not Activity views?

I wonder if there's anywhere good we can comment about this that isn't likely to go out of date.